src/client/app/containers/app.ts

```diff
@@ -175,10 +175,26 @@ export class AppComponent {
                     this._store.dispatch(new GitHub.UpdateGistAction(this.snippet));
                 }
                 else if (isPublic) {
-                    this._store.dispatch(new GitHub.SharePublicGistAction(this.snippet));
+                    let sharePublicGist = async() => {
+                        let result = await this._effects.alert(Strings.sharePublicSnippetConfirm, `${Strings.share} ${this.snippet.name}`, `${Strings.share}`, `${Strings.cancelButtonLabel}`);
```

Comment:
Strings.share is already a string, don't think you need to embed it into another string such as `${Strings.share}`